tests/e2e/bento_server_general_features/service.py

```diff
@@ -1,7 +1,7 @@
 import typing as t
 
 import numpy as np
-import pandas as pd
+import pandas as pd  # type: ignore[import]
```

Comment:
@bojiang I think `pyright` won't be able to recognize this since `import` warning is a `mypy` warning

Reply:
@aarnphm It works for my pyright 1.1.201